### ISSUE TITLE
Is there any similar project for Linux?

### ISSUE THREAD

**Issue by Ashdemai** [T800G/CBX-Shell#8]
I've been using this project when I was still using Windows and I still miss this functionality now that I'm on Linux. I've tried looking for something similar with no results. If anyone knows of something, I'd love to know about it. Thanks.

Sorry for asking this in an issue, I don't know where else to ask it.

**Comment by Mrfiregem** [comment]
This is already built in depending on what file manager you use. If you use the Gnome file manager, for example, it shows the cover (might need to have Evince installed, though)

**Comment by Ashdemai** [comment]
> This is already built in depending on what file manager you use. If you use the Gnome file manager, for example, it shows the cover (might need to have Evince installed, though)

Yes I use Nautilus the Gnome file manager, but I haven't seen a setting for showing covers for an archive file.
What is Evince? I might check it out.

**Comment by I3ordo** [comment]
Been using this in windows since 2010 and today i noticed , the thumbnails dont show up if the rar file version is 5, rars with version 4 work fine. was wondering if there is an updated version that suupprts the rar5 in this git hub but i hve no idea how to compile setups, i  forgot

**Comment by Ashdemai** [comment]
> Been using this in windows since 2010 and today i noticed , the thumbnails dont show up if the rar file version is 5, rars with version 4 work fine. was wondering if there is an updated version that suupprts the rar5 in this git hub but i hve no idea how to compile setups, i forgot

This issue is asking for any Linux version or alternatives. You might need to add your own issue if you want someone to answer you about the new version for rar.



> This is already built in depending on what file manager you use. If you use the Gnome file manager, for example, it shows the cover (might need to have Evince installed, though)

I check what Evince was, and it's just the default PDF viewer that comes installed in a base GNOME DE. I already have it. I don't think what it does is what I'm looking for.

**Comment by Mrfiregem** [comment]
Linux file managers mostly use external programs called thumbnailers to generate images based on file type. The Evince package provides one that makes thumbnails for pdf and cb* files, among others. You could make one yourself, too, if you don't want to install it.

**Comment by Ashdemai** [comment]
It's a document viewer, not a thumbnailer. I don't understand how it makes thumnbnails.
![Screenshot from 2022-10-24 04-02-49](https://user-images.githubusercontent.com/74661175/197428406-17587ff3-82b2-4428-b237-9bbc7a99944a.png)

Edit: I don't know which program is taking the first image in an archive and makes it the thumnbnail. This is why I'm asking.

**Comment by Mrfiregem** [comment]
`/usr/bin/evince-thumbnailer` would make the thumbnails if you have Evince installed.

**Comment by Ashdemai** [comment]
> `/usr/bin/evince-thumbnailer` would make the thumbnails if you have Evince installed.

I have it installed but it's not doing what I want.
